config.js

```diff
@@ -0,0 +1,100 @@
+export const config = {
+    user: {
+        getUser: {
+            name: 'getUser',
```

Comment:
No need for this name: 'getUser' key-pair, we already know from paren object what is this related to.